Updated Adobe Admin Console-related instructions and screenshots

trikay opened the pull request:
Updated some screenshots for the Adobe Admin Console and added a few links. I didn't have push access to the repo so was unable to upload the image files, so have linked to them directly from helpx.adobe.com.

azurecla commented:
Hi __@DavidRouyer__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        This seems like a small (but important) contribution, so no contribution license agreement is required at this point. Real humans will now evaluate your PR.
    </span>

TTYL, AZPRBOT;

davidwrede commented:
Hi @DavidRouyer, looks like you need to do another sync with master before we can accept this PR.

Thanks,
Dave

trikay committed:
Updated Adobe Admin Console-related instructions and screenshots

Updated some screenshots for the Adobe Admin Console and added a few links. I didn't have push access to the repo so was unable to upload the image files, so have linked to them directly from helpx.adobe.com.
